gdc_ng_models/models/misc.py

```diff
@@ -30,5 +29,8 @@ class FileReport(Base):
     report_data = Column(JSONB, nullable=True)
 
     __table_args__ = (
+        PrimaryKeyConstraint("id","timestamp"),
```

Comment:
It looks like we're mixing two ways of declaring a primary key in sqlalchemy. Can we add `primary_key=True,` to the `timestamp = Column(...` parameters instead of calling this function?

Reply:
The convention we established for the StudyRule and ReleasedData models was to use `PrimaryKeyConstraint` instead of the `primary_key` flag so that we can set an explicit primary key index name: https://github.com/NCI-GDC/gdc-ng-models/pull/18#discussion_r409743733. However, I am not sure how that interacts with declarative partitioning.

Reply:
Can a 'primary_key' be set to 'id' **and** 'timestamp'?